<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>invetory_edit</class>
 <widget class="QWidget" name="invetory_edit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>779</width>
    <height>601</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="attack_title">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Attacks and Speellcasting</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="atk_name_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>55</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="atk_bonus_label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>90</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Atk Bonus</string>
   </property>
  </widget>
  <widget class="QLabel" name="atk_dmg_label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>90</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>DMG/Type</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_atk_button">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>60</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QLabel" name="add_atk_label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>60</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Add new</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="feats_traits_title">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>300</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Features &amp; Traits</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="add_feat_button">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>340</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QLabel" name="add_feat_label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>340</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Add new</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="add_equipment_button">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>80</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QLabel" name="add_equipment_label">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>80</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Add new</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="equipment_label">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>40</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Equipment</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="currency_title">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>280</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Currency</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QFrame" name="currency_frame">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>330</y>
     <width>161</width>
     <height>161</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>10</number>
   </property>
   <property name="midLineWidth">
    <number>10</number>
   </property>
   <widget class="QLabel" name="cp_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>CP:</string>
    </property>
   </widget>
   <widget class="QLabel" name="sp_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>SP:</string>
    </property>
   </widget>
   <widget class="QLabel" name="ep_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>EP:</string>
    </property>
   </widget>
   <widget class="QLabel" name="gp_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>GP:</string>
    </property>
   </widget>
   <widget class="QLabel" name="pp_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>PP:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="cp_box">
    <property name="geometry">
     <rect>
      <x>71</x>
      <y>10</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="sp_box">
    <property name="geometry">
     <rect>
      <x>71</x>
      <y>40</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="ep_box">
    <property name="geometry">
     <rect>
      <x>71</x>
      <y>70</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="gp_box">
    <property name="geometry">
     <rect>
      <x>71</x>
      <y>100</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="pp_box">
    <property name="geometry">
     <rect>
      <x>71</x>
      <y>130</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="submit_button">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>540</y>
     <width>271</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
